docs/teams.md

```diff
@@ -6,7 +6,7 @@ You are also able to create your own team implementations by creating classes th
 
 ### ApplicationTeam 
 
-To create an `ApplicationTeam` for your cluster, simplye implement a class that extends `ApplicationTeam`. You will need to supply a team name and an array of users.  
+To create an `ApplicationTeam` for your cluster, simply implement a class that extends `ApplicationTeam`. You will need to supply a team name, an array of users, and (optionally) a directory where the team's network policy manifests exist (yaml files). **NOTE:** The yaml files are not checked for accuracy, and must be ensured that the namespace provided must match the name provided, i.e. `team-awesome`. 
```

Comment:
We don't perform any checks with respect to whether the yamls in the directory are network policies, correct? So those could be OPA policies, Kyverno and just random manifests. In that case, let's remove "where the teams' network policy manifests exist" and replace with "where you may optionally place any policy definitions and generic manifests for the team. These manifest will be applied by the platform and will be outside of the team control.".

Reply:
How is it resolved? I see `where the team's network policy manifests exist`. This directory is not for network policies, but it is for any additional generic manifests which may contain network policies.

Reply:
This has been corrected under the new PR: https://github.com/aws-quickstart/ssp-amazon-eks/pull/185